src/openzaak/components/besluiten/api/viewsets.py

```diff
@@ -449,3 +451,9 @@ def perform_create(self, serializer):
                 bio["url"] for bio in serializer.data["besluitinformatieobjecten"]
             ],
         )
+
+        process_cloudevent(
+            type=BESLUIT_VERWERKT,
```

Comment:
Could you document these newly added cloud events in `docs/api/experimental.rst`?